Testing/Unit/Rego/Teams/TeamsConfig_05-test.rego

```diff
@@ -273,17 +276,17 @@ test_PrivateCatalogAppsType_Incorrect_V1 if {
 
     count(RuleOutput) == 1
     not RuleOutput[0].RequirementMet
-    RuleOutput[0].ReportDetails == "1 meeting policy(ies) found that allow custom apps by default: Global"
+    RuleOutput[0].ReportDetails == "1 meeting policy(ies) found that does not restrict installation of third-party apps by default: Global"
```

Comment:
```suggestion
    RuleOutput[0].ReportDetails == "1 meeting policy(ies) found that does not restrict installation of custom apps by default: Global"
```

Reply:
good catch, fixed along with the other two regarding policy 5 unit test